theme/static/css/hydroshare_core.css

```diff
@@ -4256,6 +4273,22 @@ div.fb-upload-caption a {
 	font-size: 18px;
 }
 
+.fb-view-list .fb-stack {
+	width: 18px;
+	height: 18px;
+	margin-right: 10px;
+	margin-left: 10px;
+}
+
+.fb-view-list .fb-stack-database .fa-database {
+	font-size: 10px;
+    top: -4px;
+}
+
+.fb-view-list .fb-stack .fa-file-o {
+	font-size: 18px;
```

Comment:
@mauriel133 Same here, duplicate definition of style of font-size for fb-view-list here and above which I assume the later definition would override the earlier definition?